src/cljs/doit/views.cljs

```diff
@@ -102,6 +101,15 @@
         [:i {:class "fas fa-plus"}]
         "Add Todo List"]])))
 
+(defn editable-list-name [id]
+  (let [todo-list (rf/subscribe [::subs/todo-list id])]
+    (fn []
+      [:input.list-title {:type      "text"
+                          :value     (:name @todo-list)
+                          :on-change (fn [val]
+                                       (let [new-name (.-value (.-target val))]
+                                         (rf/dispatch [::events/update-todo-list (assoc @todo-list :name new-name)])))}])))
```

Comment:
This event should be dispatched when you lose focus of the `input` element. Doing this here would result in lots of unnecessary API calls.